test/make-auto.d/builder_to_archive.bats.in

```diff
@@ -4,7 +4,7 @@
     %(need_docker)s
```

Comment:
This line will need to be removed since we're running it for all builders now.
Also manually check that the test runs for all the builders.